lobster/tools/codebeamer/codebeamer.py

```diff
@@ -193,6 +215,27 @@ def to_lobster(cb_config, cb_item):
         text      = None,
         status    = status)
 
+    if cb_config.get('references'):
+        for reference_name, displayed_chosen_names in cb_config['references'].items():
+            if reference_name not in map_reference_name_to_function.keys():
+                continue
+
+            for displayed_name in displayed_chosen_names:
+                if cb_item.get(displayed_name):
+                    flat_values_list = cb_item.get(displayed_name) if isinstance(cb_item.get(displayed_name), list) \
+                        else [cb_item.get(displayed_name)]
+                else:
+                    flat_values_list = list(value for custom_field in cb_item["customFields"]
+                                            if custom_field["name"] == displayed_name and custom_field.get("values")
+                                            for value in custom_field["values"])
+                if not flat_values_list:
+                    continue
+
+                map_reference_name_to_function[reference_name](req, flat_values_list)
+
+    if "name" not in cb_item:
+        req.error("Item lacks a summary text")
```

Comment:
This would re-introduce the limitation to allow only items with a summary. Please remove it.